packages/adminrouter/docker/requirements-tests.txt

```diff
@@ -3,7 +3,7 @@ PyJWT==1.5.3
 cryptography>=1.3.4
 dnslib==0.9.7
 flake8-isort==2.2.2
-flake8==3.4.1
+flake8==3.7.9
```

Comment:
I wanted to improve the code I was working with a bit, leave it better than it was. I had to do some linting fixes to my code anyway and decided that it will not hurt if I do it using the latest flake8.